user-guide/Reference/Metrics/user-defined_API_benchmarks.md

```diff
@@ -0,0 +1,28 @@
+---
+uid: user-defined_API_benchmarks
+---
+
+# User-defined API benchmarks
+
+## Specifications of the test VM server 1
```

Comment:
Not sure why a "1" is added if only one server is mentioned. I'd suggest to keep it simple:

```suggestion
## Specifications of the test VM server
```